.ui/pgHome.pgx

```diff
@@ -87,43 +68,6 @@
 				"testId": "Z--GuhP9L",
```

Comment:
we shouldn't be using user props inside page

Reply:
done.